cmd/server/main.go

```diff
@@ -1,5 +1,5 @@
 /*
-Package main is the core service layer for the BUX Server
+Package main is the core service layer for the SPV Wallet
 */
 package main
```

Comment:
Maybe it's a good time to update directory from:
~~cmd/server/main.go~~
to
**cmd/main.go**

Reply:
directory updated